src/lib/ui/components/File/File.svelte

```diff
@@ -45,8 +45,11 @@
           ? ([...files, ...filesToUpload] as File[])
           : filesToUpload[0];
 
-      if (fileToUpload instanceof File) {
-        const sizeKB = fileToUpload.size / 1024;
+      if (fileToUpload instanceof File || Array.isArray(fileToUpload)) {
+        const sizeKB =
+          (fileToUpload instanceof File
```

Comment:
🚫 **[eslint]** <[operator-linebreak](https://eslint.org/docs/latest/rules/operator-linebreak)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>There should be no line break before or after '='.
```suggestion
        const sizeKB =          (fileToUpload instanceof File
```